operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -37,7 +37,7 @@ type CentralSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2,displayName="Scanner Component Settings"
 	Scanner *ScannerComponentSpec `json:"scanner,omitempty"`
 
-	// Settings for the Scanner V4 component, which can run in addition to the previously existing Scanner components
+	// Settings for the Scanner V4 components, which can run in addition to the previously existing Scanner components
```

Comment:
Maybe text after `//` is internal and not visible to users, but I'll mark suggestions and you can ignore if not applicable.

Reply:
I've changed all occurrences of Scanner v4 to Scanner V4.

Just FYI: If a comment `//` is directly above a field in a struct in `central_types.go` `securedcluster_types.go` or `common_types.go` you can assume it will be visible in the operator UI.